src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -247,6 +247,83 @@
  *         }
  *     }
  *
+ * ## Tokens and finer-grained processing.
+ *
+ * It's also possible to scan through every lexical token in
+ * the HTML document using the `next_token()` function. This
+ * alternative form takes no argument and provides no built-in
+ * query syntax.
```

Comment:
NIT: The also here seems to link closely to the last section, but I don't understand the close relationship. I'd let this stand on its own:

```suggestion
 * It's possible to scan through every lexical token in the
 * HTML document using the `next_token()` function. This
 * alternative form takes no argument and provides no built-in
 * query syntax.
```

Reply:
thanks. I adopted your suggestion in 8793d47fdec323539ccfc58efb4103fb7a864a6d